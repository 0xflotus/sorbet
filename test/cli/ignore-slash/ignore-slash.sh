#!/bin/bash

# Tests both relative and absolute ignore patterns.
main/sorbet --silence-dev-message --ignore foo/ test/cli/ignore-slash 2>&1
