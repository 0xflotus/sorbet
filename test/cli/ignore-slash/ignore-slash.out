test/cli/ignore-slash/bar.rb:4: `String("bar.rb")` doesn't match `Integer` for argument `arg0` http://go/e/7002
     4 |1 + "bar.rb"
        ^^^^^^^^^^^^
    https://git.corp.stripe.com/stripe-internal/sorbet/tree/master/rbi/core/integer.rbi#L103: Method `Integer#+` has specified `arg0` as `Integer`
     103 |        arg0: Integer,
                  ^^^^
  Got String("bar.rb") originating from:
    test/cli/ignore-slash/bar.rb:4:
     4 |1 + "bar.rb"
            ^^^^^^^^
Errors: 1
